modules/oe_content_persistent/config/install/oe_content_persistent.settings.yml

```diff
@@ -0,0 +1,2 @@
+langcode: en
+inter_institutional_domain: 'https://data.ec.europa.eu/ewp/'
```

Comment:
Let's call this a base path, not domain. Since there is more than just the domain configured :)

Reply:
done
didn't saw comment. currently I renamed to `inter_institutional_base_url`. Is this ok?

Reply:
renamed exactly to `base_url`